hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -632,6 +637,7 @@ static boolean validateConfigVersion(ConfigProperty<?> configProperty, HoodieTab
     // validate that the table version is greater than or equal to the config version
     HoodieTableVersion firstVersion = HoodieTableVersion.fromReleaseVersion(configProperty.getSinceVersion().get());
     boolean valid = tableVersion.greaterThan(firstVersion) || tableVersion.equals(firstVersion);
+    valid = valid || CONFIGS_REQUIRED_FOR_OLDER_VERSIONED_TABLES.contains(configProperty.key());
```

Comment:
What is the rationale here?  It looks like we need to validate this in another place instead of here.

Reply:
@lokeshj1703 : can you respond to this.

Reply:
These configs are added with since version 1.0.0 but are still required for table version 6. Therefore I had to add a check here so that these configs are not dropped for tables with version 6.